pkg/action/build-uki.go

```diff
@@ -49,47 +55,72 @@ func (b *BuildUKIAction) Run() error {
 		return err
 	}
 
-	b.logger.Info("extracting image to a temporary directory")
+	b.logger.Info("Extracting image to a temporary directory")
 	sourceDir, err := b.extractImage()
 	if err != nil {
 		return err
 	}
 	defer os.RemoveAll(sourceDir)
 
-	b.logger.Info("creating additional directories")
+	b.logger.Info("Creating additional directories in the rootfs")
 	if err := b.setupDirectoriesAndFiles(sourceDir); err != nil {
 		return err
 	}
 
-	b.logger.Info("creating an initramfs file")
+	b.logger.Info("Creating an initramfs file")
 	if err := b.createInitramfs(sourceDir); err != nil {
 		return err
 	}
 
 	cmdlines := utils.GetUkiCmdline()
 	for _, cmdline := range cmdlines {
-		b.logger.Info("running ukify for cmdline: " + cmdline)
+		b.logger.Info("Running ukify for cmdline: " + cmdline)
 		if err := b.ukify(sourceDir, cmdline); err != nil {
 			return err
 		}
-		b.logger.Info("creating kairos and loader conf files")
+		b.logger.Info("Creating kairos and loader conf files")
 		if err := b.createConfFiles(sourceDir, cmdline); err != nil {
 			return err
 		}
 	}
 
-	b.logger.Info("running sbsign")
+	b.logger.Info("Signing artifacts")
 	if err := b.sbSign(sourceDir); err != nil {
 		return err
 	}
 
-	if err := b.createISO(sourceDir); err != nil {
-		return err
+	switch b.artifact {
+	case string(constants.IsoOutput):
+		err = b.createISO(sourceDir)
+		b.logger.Infof("Done building %s at: %s", b.artifact, b.outputDir)
+	case string(constants.ContainerOutput):
+		// First create the files
+		err = b.createArtifact(sourceDir)
+		if err != nil {
+			return err
+		}
+		// Then build the image
+		kairosVersion, _ := findKairosVersion(sourceDir)
```

Comment:
shouldn't we fail if we don't find the version here? as it actually output files are named after it

Reply:
yes!